msresist/data/PSPL/pssm_data.csv

```diff
@@ -142,22 +142,6 @@ ERK1,,,,,,,,,,,,,,,,,,,,
 2,2,1.3,1,1,2,1,1,1.2,1.5,1,1,1,1.3,5,1,1,1,1,1,1
 2,1,1,1,1,1,1,1,1.2,1,1.2,1.3,1,1.3,2,1.5,1.5,1.2,1.3,1.2,1
 2,1,1,1,1,1,1,1,1.2,1,1.2,1.3,1,1.3,1.2,1.5,1.5,1.2,1.3,1.2,1
-PKD1,,,,,,,,,,,,,,,,,,,,
-1,1,1,1,1,1,1,2,4,4,2,1,2,2,2,1,1,1,2,2,1
-1,1,1,1,1,1,1,2,1,4,2,1,2,2,4,1,1,1,2,4,1
-1,3,10,1,2,18,1,1,1,1,1,1,1,2,1,1,1,1,2,1,1
-1,3,6,1,2,6,1,1,1,4,1,1,1,2,1,1,1,1,2,1,1
-1,1,1,1,1,1,1,3,1,1,1,1,3,2,1,1,1,1,2,4,1
-1,1,1,1,3,1,1,3,1,1,2,3,3,2,1,3,4,5,2,5,1
-1,2,2,1,5,4,1,3,1,1,3,1,3,2,1,1,1,1,2,5,1

```

Comment:
I assume you wanted to delete these?

Reply:
Yes, we already have PKD from the previous collection of PSPL data I imported and this SH2 is repeated in this data set for some reason. I want to keep the ipynb to do this analysis separate for each phosphorylation type.